command/rubycommand/rubycommand.go

```diff
@@ -213,3 +214,55 @@ func IsGemInstalled(gem, version string) (bool, error) {
 
 	return findGemInList(out, gem, version)
 }
+
+func isSelectedRbenvRubyInstalled(message string) (bool, string, error) {
+	//
+	// Not installed
+	reg, err := regexp.Compile("rbenv: version \x60.*' is not installed") // \x60 == ` (The go linter suggested to use the hex code instead)
+	if err != nil {
+		return false, "", fmt.Errorf("failed to parse regex ( %s ) on the error message, error: %s", "rbenv: version \x60.*' is not installed", err) // \x60 == ` (The go linter suggested to use the hex code instead)
+	}
+
+	var version string
+	if reg.MatchString(message) {
+		message := reg.FindString(message)
+		version = strings.Split(strings.Split(message, "`")[1], "'")[0]
+		return false, version, nil
+	}
+
+	//
+	// Installed
+	reg, err = regexp.Compile(".* \\(set by")
+	if err != nil {
+		return false, "", fmt.Errorf("failed to parse regex ( %s ) on the error message, error: %s", ".* \\(set by", err)
+	}
+
+	if reg.MatchString(message) {
+		s := reg.FindString(message)
+		version = strings.Split(s, " (set by")[0]
+		return true, version, nil
+	}
+	return false, version, nil
+}
+
+// IsSelectedRbenvRubyInstalled checks if the selected ruby version is installed via rbenv.
+// Ruby version is set by
+// 1. The RBENV_VERSION environment variable
+// 2. The first .ruby-version file found by searching the directory of the script you are executing and each of its
+// parent directories until reaching the root of your filesystem.
+// 3.The first .ruby-version file found by searching the current working directory and each of its parent directories
+// until reaching the root of your filesystem.
+// 4. The global ~/.rbenv/version file. You can modify this file using the rbenv global command.
+func IsSelectedRbenvRubyInstalled(workdir string) (bool, string, error) {
```

Comment:
NOTE: For me it is very strange that it returns `installed` for the version value given as an input param, however returns a version number that is currently installed.

I think that a logic like:
- getting the currently installed one
- getting a version from the ruby-version file
- comparing them in a main thread

might be more clean and understandable, WDYT?

Reply:
I removed the version from the return parameters as we don't need it, but I don't want to check the files/ envs, etc manually. Just let the Rbenv to do it.